doc/manual/manual.tex

```diff
@@ -2311,7 +2329,7 @@ \subsection{Calculations with melt transport}
   \\
   &\quad
   - \frac{\phi }{\rho_f} \mathbf{u}_s \cdot \nabla\rho_f
-  - (\mathbf{u}_s \cdot \mathbf g ) (1 - \phi) \kappa_s \rho_s
+  - (\mathbf{u}_s \cdot \mathbf g ) (1 - \phi) \beta_s \rho_s
```

Comment:
Can you add a note somewhere that in the paper that describes the implementation (`\cite{dannberg_melt}`), we use kappa for the compressibility, while here we use beta to be consistent throughout the manual.

Reply:
Done.